src/psyclone/transformations.py

```diff
@@ -325,6 +325,9 @@ class OMPTaskloopTrans(ParallelLoopTrans):
     :type grainsize: int or None
     :param num_tasks: the num_tasks to use for this transformation.
     :type num_tasks: int or None
+    :param nogroup: Whether of not to use a nogroup clause for this
```

Comment:
s/Whether/whether/ please

Reply:
Done - also removed another capital letter that had escaped previous reviews in OMP Single